pkg/porter/testdata/show/expected-output.json

```diff
@@ -1,6 +1,7 @@
 {
   "schemaType": "Installation",
   "schemaVersion": "1.0.0",
+  "id": "01FZVC5AVP8Z7A78CSCP1EJ604",
```

Comment:
This is just a heads up that we have a follow-up issue to expose this value as an environment variable and template value in the bundle. I think it will be useful for people to have a (pretty) random value to use with their installation when setting resource names.